JordanRosas/Contact-List
# Contact list 1

completed contact list practice from book 4. utilizing browserify.

steps to test:
1. `git fetch --all`
2. `git checkout contact-List-1`
3. navigate to the lib directory in the terminal
4.run grunt
5. in the browser go to `localhost:8080`
6. fill out the fields named: `name. number, address` and click save
7. verify that the page refreshes when button is clicked 
8. verify that the user contact is displayed in the browser
9. open up VS code and verify that the database JSON file is updated with the user input as well
